ui/src/components/integrationFormFields/integrationFormFields.tsx

```diff
@@ -23,6 +23,17 @@ export const IntegrationFormFields: FC<IntegrationFormFieldsInterface> = (props)
     });
   }, []);
 
+  const projectIdValidator = (value: string) => {
+    const trimmedValue = value ? value.trim() : value;
+    if (trimmedValue === '' || trimmedValue === undefined || value.length > PROJECT_ID_MAX_LENGTH) {
```

Comment:
Looks like these 2 checks trimmedValue === '' || trimmedValue === undefined refer to required field and we can use this validator.